### PR TITLE
Fix parameters so things actually work in Dashing.

### PR DESCRIPTION
**Pull request by clalancette**
Signed-off-by: Chris Lalancette <clalancette@openrobotics.org>

### CODE REVIEW

**Review by jubeira — COMMENTED** [review]
I did a quick pass and it looks good; +1 to using move for Twist messages.
Please feel free to get a review from one of the regular maintainers if you consider it necessary.

**Review by mjcarroll — APPROVED** [review]

**Pull request merged**
